freedom-metal/core_portme.mak

```diff
@@ -32,8 +32,8 @@ OUTFLAG= -o
 #AS		= gas
 # Flag : CFLAGS
 #	Use this flag to define compiler options. Note, you can add compiler options from the command line using XCFLAGS="other flags"
-PORT_CFLAGS = -O3 -DITERATIONS=3000 -DPERFORMANCE_RUN=1 -DMAIN_HAS_NOARGC=1 -DMAIN_HAS_NORETURN=1 -DHAS_STDIO -DHAS_PRINTF -DHAS_TIME_H -DUSE_CLOCK
-FLAGS_STR = "$(PORT_CFLAGS) $(XCFLAGS) $(XLFLAGS) $(LFLAGS_END)"
+PORT_CFLAGS = -O3 -DITERATIONS=3000 -DPERFORMANCE_RUN=1 -DMAIN_HAS_NOARGC=1 -DMAIN_HAS_NORETURN=1 -DHAS_STDIO -DHAS_PRINTF -DHAS_TIME_H -DUSE_CLOCK -DHAS_FLOAT=0
+FLAGS_STR = " "
```

Comment:
Was checking with Palmer, and still want it display for FPGA run.
Can you add this as an option in the settings.mk. That is fo RTL nothing to be print, for FPGA print these.

Reply:
Hi Bunnaroath, I've rerun the regression on both RTL and ARTY. We can pass the regression and get the same scores as reported. We can keep the FLAG output for all configs.